contrib/fonttools/showttf.c

```diff
@@ -2887,6 +2887,7 @@ static void gdefshowligcaretlist(FILE *ttf,int offset,struct ttfinfo *info) {
 		printf( "!!!! Unknown caret format %d !!!!\n", format );
 	    }
 	}
+        free(offsets);
```

Comment:
I checked on my local repo, I've got this:
   2886             } else {
   2887                 printf( "!!!! Unknown caret format %d !!!!\n", format );
   2888             }
   2889         }
   2890         free(offsets);
Perhaps it's just due to the "+"?